test/utils/database-state-creator.ts

```diff
@@ -6,10 +6,10 @@ import fs from "fs";
 import path from "path";
 
 interface CreatorResult {
-	installation: Installation,
-	subscription: Subscription,
-	gitHubServerApp?: GitHubServerApp,
-	repoSyncState?: RepoSyncState
+	installation: Installation | null;
+	subscription: Subscription | null;
```

Comment:
My understanding is that installation and subscription must always be defined. Should  we maybe use "!" instead of " |null"? Otherwise, all the users of this class will suffer.... 😬

Reply:
Same deal here. We're calling `findByPk`:

```
installation: await Installation.findByPk(installation.id),
subscription: await Subscription.findByPk(subscription.id),
gitHubServerApp: gitHubServerApp ? await 
  GitHubServerApp.findByPk(gitHubServerApp.id) : undefined, 
repoSyncState: repoSyncState ? await RepoSyncState.findByPk(repoSyncState.id) : undefined`